.clang-format

```diff
@@ -36,12 +36,13 @@ IncludeCategories:
   # 1. Related header
   # 2. All private headers
   # 3. All public headers from this repository (maya-usd)
-  # 4. Pixar + USD headers
-  # 5. Autodesk + Maya headers
-  # 6. Other libraries' headers
-  # 7. C++ standard library headers
-  # 8. C system headers
-  # 9. Conditional includes
+  # 4. UsdUfe headers
```

Comment:
Added new section for the UsdUfe headers. Running clang-format locally moved all of them to this section.